lib/datamodel/datamodel.js

```diff
@@ -1,14 +1,50 @@
-import { models } from '../../models/init-models.js'
-import { Graph } from './graph.js'
-import sequelizeConn from '../../util/db.js'
-import { getTableNameByNumber } from '../table-number.js'
+import { models } from "../../models/init-models.js"
+import { Graph } from "./graph.js"
+import { Edge } from "./edge.js"
+import sequelizeConn from "../../util/db.js"
+import { getTableNameByNumber } from "../table-number.js"
 
 export class Datamodel {
+
+  static datamodel = null
+  
   constructor() {
     this.graph = new Graph()
+  }
+
+  initialize() {
+    // Add tables
     for (const model in models) {
       this.graph.addNode(sequelizeConn.models[model])
     }
+    // Add relationships
+    for (const model in models) {
+      const table = sequelizeConn.models[model]
+      for (const [field, attributes] of Object.entries(table.rawAttributes)) {
+        if (attributes.references) {
+          const neighborName = attributes.references.model
+          const neighbor = this.getTableByName(neighborName)
```

Comment:
`this.getTableByName` and `this.tableExists` are instance methods that do not exists in this class.

Reply:
`this.getTableByName` is defined on line 85 and `this.tableExists` is defined on line 56 of this class?